Makefile

```diff
@@ -39,6 +40,15 @@ ICON_NANOX = icons/nano_app_eos.gif
 ICON_NANOSP = icons/nano_app_eos.gif
 ICON_STAX = icons/stax_app_eos.gif
 ICON_FLEX = icons/flex_app_eos.gif
+ICON_APEX = icons/apex_app_eos.gif
+
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_NANOS2))
+    # With the Nano NBGL Design, the Home Screen icon is the reverse of the App icon:
+    # It should be on white background, with rounded corners.
+    # This definition allows SDK Makefiles to automatically generate it based on the App icon.
+    # Please note that the icon is dynamically generated, and declared in the .gitignore to avoid storing it.
+    ICON_HOME_NANO = glyphs/home_app_eos.gif
+endif
```

Comment:
I would prefer if we use this once for generating the home glyph then remove this whole snippet and commit the actual used glyph